src/site/content/en/blog/wakelock/index.md

```diff
@@ -111,65 +111,52 @@ that your app can continue running.
-the first parameter, which *currently* is limited to just `'screen'`.
-You also need a way to abort the wake lock, which works through the
-generic `AbortController` interface. Therefore, you first create a new
-[`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController),
-and then pass the controller's
-[`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal)
-as the second parameter to `WakeLock.request()`. Two things can happen next
-that you need to `catch`:
-
-* The wake lock can, after a while, just be regularly aborted, which you
-  detect by checking if the exception's name is `'AbortError'`. In this
-  context, `AbortError` is actually not an error in the common sense, but
-  just the way `AbortController` works.
-* The browser can also refuse the request for various reasons (for example,
-  because the battery charge level is too low). In this case, the exception's
-  message will contain more details.
+To request a wake lock, you need to call the `navigator.wakeLock.request()` method
+that returns a `WakeLockSentinel` object.
+You pass this method the desired wake lock type as a parameter,
+which *currently* is limited to just `'screen'`.
+The browser can refuse the request for various reasons (for example,
+because the battery charge level is too low),
+so it's a good practice to wrap the call in a `try…catch` statement.
+The exception's message will contain more details in case of failure.
+
+You also need a way to release the wake lock, which works through the
+`release()` method of the `WakeLockSentinel` object.
+If you don't store a reference to the `WakeLockSentinel`, there's no way
+to release the lock manually, but it will be released once the current tab is invisible.
 
 ```js
-const wakeLockCheckbox = document.querySelector('#wakeLockCheckbox');
-
-if ('WakeLock' in window) {
-  let wakeLock = null;
-
-  const requestWakeLock = () => {
-    const controller = new AbortController();
-    const signal = controller.signal;
-    window.WakeLock.request('screen', {signal})
-        .catch((e) => {
-          if (e.name === 'AbortError') {
-            wakeLockCheckbox.checked = false;
-            console.log('Wake Lock was aborted');
-          } else {
-            console.error(`${e.name}, ${e.message}`);
-          }
-        });
-    wakeLockCheckbox.checked = true;
+// The wake lock sentinel.
+let wakeLock = null;
+
+// Function that attempts to request a wake lock.
+const requestWakeLock = async () => {
+  try {
+    wakeLock = await navigator.wakeLock.request('screen');
+    wakeLock.addEventListener('release', () => {
+      console.log('Wake Lock was released');
+    });
     console.log('Wake Lock is active');
-    return controller;
-  };
-
-  wakeLockCheckbox.addEventListener('change', () => {
-    if (wakeLockCheckbox.checked) {
-      wakeLock = requestWakeLock();
-    } else {
-      wakeLock.abort();
-      wakeLock = null;
-    }
-  });
-}
+  } catch (e) {
```

Comment:
Can we use `err` instead of `e` to avoid confusing the variable for something that holds an event?

Reply:
Thanks, fixed!